### ISSUE TITLE
Regarding the issue of outputting probes data

### ISSUE THREAD

**Issue by HanWang1893** [code-mphi/ECOGEN#43]
Dear all,
 
I need to output the subtle physical quantity changes on a certain line in the 2D example, so I set up a series of probes and fixed the output time frequency. However, when I was processing the output results, I found that the array length of each point record seemed to be different, which made it impossible for me to transpose the matrix. This phenomenon only occurs in my recent examples, and I speculate that it may be on an approximate order of magnitude with the time interval between the calculation time step I set and the output point?

 Can you tell me how to solve the problem of inconsistent array length when outputting results from probes at different positions? Or other methods can directly output data on a custom location line.

![image](https://github.com/code-mphi/ECOGEN/assets/128766769/bf748486-1312-4c4b-afa9-8dd0b8f3e2e5)

Thanking you in advance
HanWang

**Comment by kevinschmidmayer** [comment]
Dear HanWang,

I'm sorry I only realise now your issue. Also, I was not able to reproduce your problem. Did you find a solution? Or would you be willing to share your input files so we could eventually reproduce the problem on our side?

Best,
Kevin

**Comment by HanWang1893** [comment]
Dear Kevin,

Sorry, it has been a long time and I have deleted the example where this problem occurred. It may just be an occasional error or my operational mistake. I have checked the currently saved examples and there is no issue of inconsistent save time steps at each point. If I encounter this problem again in the future, I will upload the example configuration.

Thanking you in advance
HanWang
